app/controllers/contact_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ContactController < ApplicationController
+  def new
+    @contact = Contact.new
+  end
+
+  def create
+    @contact = Contact.new contact_params
+
+    if @contact.valid?
+      ContactMailer.contact_us(@contact).deliver_now
+      redirect_to new_contact_url, notice: 'Message received, thanks!'
```

Comment:
The most common message in contact pages is "The message has been send.....", what do you think?

Reply:
But It must be `The message has been sent`